app/pages/Recovery/DisplayRecovery.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+    accountsInfoSelector,
+    loadAccountInfos,
+} from '~/features/AccountSlice';
+import { Account } from '~/utils/types';
+import AccountCard from '~/components/AccountCard';
+import CardList from '~/cross-app-components/CardList';
+import { Status } from './util';
+
+import styles from './Recovery.module.scss';
+
+interface Props {
+    status?: Status;
+    recoveredAccounts: Account[][];
+}
+
+/**
+ * Column, which displays the recovered accounts and the current status.
+ */
+export default function DisplayRecovery({ status, recoveredAccounts }: Props) {
+    const dispatch = useDispatch();
+    const accountsInfo = useSelector(accountsInfoSelector);
+
+    useEffect(() => {
+        if (recoveredAccounts.length) {
+            loadAccountInfos(
+                recoveredAccounts[recoveredAccounts.length - 1],
+                dispatch
+            );
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [recoveredAccounts.length]);
+
+    return (
+        <div className={styles.recoveredDiv}>
+            {recoveredAccounts.map((accounts, index) => (
+                <>
+                    <p className="bodyEmphasized textLeft">Index {index}:</p>
+                    <p className="textLeft">
```

Comment:
In the Figma sketch there is not a newline between the line above and this one.

Reply:
I have reduced the space between the two lines.